app/models/user.rb

```diff
@@ -1,7 +1,8 @@
 class User < ApplicationRecord
-  attr_accessor :remember_token
+  attr_accessor :remember_token, :activation_token
   ATTR_CHANGE = %i(name email password password_confirmation).freeze
   before_save :downcase_email
+  before_create :create_activation_digest
```

Comment:
```suggestion
  before_create :create_activation_digest
  
```
Em nên thêm 1 dòng trống giữa các bolck để thấy rõ hơn nhé như ở đây cần thêm 1 dòng trống để tách black call back với blonk validate